pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -190,6 +190,11 @@ func (m *kubeGenericRuntimeManager) generateContainerConfig(container *v1.Contai
 	}
 
 	command, args := kubecontainer.ExpandContainerCommandAndArgs(container, opts.Envs)
+	logDir := BuildContainerLogsDirectory(kubetypes.UID(pod.UID), container.Name)
```

Comment:
Remove the directory in `removeContainerLog`

Reply:
done